src/Groups/gsets.jl

```diff
@@ -219,13 +219,43 @@ end
 
 ## (add more such actions: on sets of sets, on sets of tuples, ...)
 
-## natural action of a permutation group on the integers 1, ..., degree
-gset(G::PermGroup) = gset(G, 1:G.deg; closed = true)
+#############################################################################
+##
+##  natural method with implicit action function
+
+"""
+    natural_gset(G::PermGroup)
+
+Return the G-set `Omega` that consists of the closure of integers 1, ..., degree
```

Comment:
```suggestion
Return the G-set `Omega` that consists of integers 1, ..., degree
```
`closed = true` says that we explicitely don't take a closure here